src/Standards/BestIt/ruleset.xml

```diff
@@ -50,4 +50,8 @@
     <rule ref="Generic.Formatting.SpaceAfterCast" />
 
     <rule ref="Squiz.Strings.DoubleQuoteUsage.NotRequired" />
+
+    <rule ref="SlevomatCodingStandard.TypeHints.PropertyTypeHint.MissingAnyTypeHint">
```

Comment:
Please overwrite it with a custom best it class and implement unit tests.

Reply:
Done